src/templates/kanban-template/kanban-template.stories.tsx

```diff
@@ -63,7 +48,7 @@ export const kanbanTemplate: Story = {
         kanbanProps={{
           ...args.kanbanProps,
           onComponentListChange: (componentList) => {
-            console.log("Component list changed", componentList);
+            action("onComponentListChange")(componentList);
           },
```

Comment:
Doesn't `onComponentListChange: action("onComponentListChange"),` achieve the same logging?